server/unit_tests/test_main.py

```diff
@@ -27,3 +27,22 @@ def test_read_get_submission():
                                            "group_run_id": 1,
                                            "run_time": "2000-10-31T01:30:00-05:00",
                                            "seed": 1}}]}
+
+
+def test_read_get_submissions():
+    response = client.get('/get_submissions/1')
+    print(response.json())
```

Comment:
Lets not leave print statements in the test, this may be done earlier, if so please remove those as well. Good (official) first contribution!